src/Schema/AST/ASTBuilder.php

```diff
@@ -87,9 +87,11 @@ public function documentAST(): DocumentAST
 
         $cacheConfig = $this->configRepository->get('lighthouse.cache');
         if ($cacheConfig['enable']) {
-            $this->documentAST = app('cache')->remember(
+            /** @var \Illuminate\Contracts\Cache\Repository $cache */
+            $cache = app('cache');
+            $this->documentAST = $cache->remember(
                 $cacheConfig['key'],
-                $cacheConfig['ttl'],
+                $cacheConfig['ttl'] ?? null,
```

Comment:
Maybe if they are missing this is because they didn't update the config. So... is it not better to let them know that and force the config update?